src/content/docs/cheerpj3/05-reference/Runtime-API.md

```diff
@@ -0,0 +1,444 @@
+
+```js
+cheerpjInit({ clipboardMode: "system" });
+```
+
+In `system` mode CheerpJ will share the clipboard with the system. Browsers enforce serious limitations on how the system clipboard can be accessed. In practice it is generally accessible when the `Ctrl+C` and `Ctrl+V` shortcuts are used (`Cmd+C` and `Cmd+V` on MacOSX). Due to these limitations the UX when using `clipboardMode:"system"` is:
+
+- `Ctrl+C`/`Cmd+C`: the user has to press the shortcut twice to give CheerpJ access to the system clipboard. CheerpJ will block the execution while waiting for the second `Ctrl+C`.
+- `Ctrl+V`/`Cmd+V`: this shortcut behaves normally, there is no difference with native execution.
+- Menu based Copy/Paste: `Ctrl+C`/`Ctrl+V` are needed to access the clipboard. CheerpJ will block the execution while waiting the appropriate shortcut.
+
+Commercial users adopting this integration have so far reported that this change in UX is not a significant burden for users. Moreover, in the future we plan to add an additional clipboard mode to take advantage of a new permission-based browser API which is currently being standardized. This future mode will provide native like user experience in all cases.
+
+#### `disableErrorReporting`
+
+CheerpJ automatically reports errors at runtime. Setting this option to `true` disables this system.
+
+Example:
+
+```js
+cheerpjInit({ disableErrorReporting: true });
+```
+
+#### `disableLoadTimeReporting`
+
+CheerpJ automatically get data about loading time. Setting this option to `true` disables this system.
+
+Example:
+
+```js
+cheerpjInit({ disableLoadTimeReporting: true });
+```
+
+#### `enableInputMethods`
+
+When this option is set to `true` CheerpJ will be able to receive text input from the input method framework of the platform. This is useful to support text input for languages such as Chinese, Japanese and Korean.
+
+```js
+cheerpjInit({ enableInputMethods: true });
+```
+
+#### `enableProguardTrace`
+
+When this option is set to `true` CheerpJ will automatically keep track of the classes actually used at runtime. Moreover, it will also keep track of classes which are accessed by reflection. After the application has been fully tested you can use the `cjGetProguardConfiguration()` function from the browser console to download a ProGuard configuration file (`cheerpj.pro`) that you can directly use with ProGuard to remove unneeded classes, methods and fields from the application, greatly reducing the download size and startup time.
+
+```js
+cheerpjInit({ enableProguardTrace: true });
+```
+
+#### `javaProperties`
+
+An array of Java properties in the form `"key=value"`. They will be defined on the System object (System properties). This option should be used if command line arguments in the form `-Dkey=value` are required when using native Java.
+
+Example usage:
+
+```js
+cheerpjInit({ javaProperties: ["prop1=value1", "prop2=value2"] });
+```
+
+#### `listener`
```

Comment:
This stuff has also changed, please review on discord, i have discussed the changes at length